server/controllers/stock/setting/index.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * The Stock Settings Controller
+ *
+ * This controller is responsible for creating and updating stock-related settings.
+ */
+
+const db = require('../../../lib/db');
+const NotFound = require('../../../lib/errors/NotFound');
+// ??? const BadRequest = require('../../../lib/errors/BadRequest');
+
+// GET /stock/setting
+//
+// Get the current stock settings for the Enterprise
+//    If req.query.enterprise_id is set, it will use that,
+//    otherwise it will look up the entry for Enterprise.id=1
+exports.list = function list(req, res, next) {
+  const enterpriseId = req.query.enterprise_id || '1';
+  const sql = `
+    SELECT month_average_consumption, default_min_months_security_stock,
+      enable_auto_purchase_order_confirmation, enable_auto_stock_accounting,
+      enable_daily_consumption, enable_strict_depot_permission,
+      enable_supplier_credit
+    FROM stock_setting
+    WHERE enterprise_id = ${enterpriseId} LIMIT 1;
+    `;
+
+  db.exec(sql)
+    .then(rows => {
+      res.status(200).json(rows);
+    })
+    .catch(next)
+    .done();
+};
+
+// POST /stock/setting
+exports.create = function create(req, res, next) {
```

Comment:
Interesting... for this, we probably don't have a create route, eh?  Do we even need this?

Reply:
Route removed